src/styles/blocks/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  background: var(--white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: 200px;
+  height: 408px;
+  padding: 16px;
+}
+
+.card__image {
+  margin: 16px 3px 40px;
+  display: block;
+  width: 160px;
+  height: 134px;
```

Comment:
to make sure the aspect ratio is preserved, you can simply set the width

Reply:
For some reason if I don't set the height, it would be only 133.203px and it fails the tests.